testing/integration-tests/src/full_client/mod.rs

```diff
@@ -2,15 +2,12 @@
 // This file is dual-licensed as Apache-2.0 or GPL-3.0.
 // see LICENSE for license details.
 
-#[cfg(test)]
+#![cfg(test)]
```

Comment:
Could we remove this? Since we have a `cfg(test) mod full_client` in `src/lib.rs`?